common/men_at_arms_types/00_maa_types.txt

```diff
@@ -169,9 +202,69 @@ onager = {
 	
 	stack = 10
 	allowed_in_hired_troops = no
+	icon = seige_weapon
 }
 
+giant_unit = {
+	type = massive_infantry
+	
+	damage = 36
+	toughness = 35
+	pursuit = 0
+	screen = 13
+	
+	terrain_bonus = {
+		wetlands = { damage = -3 pursuit = -6 }
+		jungle = { damage = -10 pursuit = -10 }
+		mountains = { damage = -5}
+
+	}
+
+	counters = {
+		heavy_infantry = 1
+		skirmishers = 1
+	}
+
+	buy_cost = { gold = massive_infantry_recruitment_cost }
+	low_maintenance_cost = { gold = massive_infantry_low_maint_cost }
+	high_maintenance_cost = { gold = massive_infantry_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = culture_ai_weight_heavy_cavalry }
+	icon = massive_infantry
+
+}
+
+
 #Medieval Era
+aerial_unit = {
+	type = light_aerial
+	
+	damage = 20
+	toughness = 12
+	pursuit = 30
+	screen = 5
+	
+	terrain_bonus = {
+		forest = { damage = -10 pursuit = -10 }
+		jungle = { damage = -10 pursuit = -10 }
+
+	}
+
+	counters = {
+		massive_aerial = 1
+		caster = 1
```

Comment:
It's weird they counter casters but countered by archers and both are ranged. It makes more sense logic-wise if they counter skirmishers.

Reply:
Ignore this, I posted the entire balance idea on Discord.